src/classes/BGE_ConfigurationWizard_CTRL.cls

```diff
@@ -191,38 +191,40 @@ public with sharing class BGE_ConfigurationWizard_CTRL {
             'EMAIL',
             'MULTIPICKLIST',
             'PHONE',
-            'URL'
+            'URL',
+            'REFERENCE'
         };
 
         // these fields have been whitelisted since they have special handling in BDI
         Set<String> allowedFields = new Set<String>{
-            'DonationCampaignImported__c'
+                'Donation_Record_Type_Name__c'
```

Comment:
This pattern is no longer needed for DonationCampaignImported__c, but is still needed for Donation Record Type, which came in a separate sibling branch. Added it here so it wasn't an empty set in this branch.